agents/utils/configs.py

```
@@ -0,0 +1,167 @@
+import asyncio
+import json
+import logging
+
+from cdp import Cdp, Wallet
```

Ruff (0.8.2)

`cdp.Wallet` imported but unused

Remove unused import: `cdp.Wallet`

(F401)